java-security-test/src/main/java/com/sap/cloud/security/test/util/FileReaderUtil.java

```diff
@@ -0,0 +1,56 @@
+package com.sap.cloud.security.test.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+/**
+ * Utility class that loads the content of files.
+ */
+public class FileReaderUtil {
```

Comment:
why do we need that? 
Can't you use IOUtils.resourceToURL("/publicKey.txt").getPath() within the buildpack?

Reply:
Yes you could do that of course. I just wanted to make it easier on the calling site. But I can remove this if you think it is not worth the extra class.

Reply:
There is no use case and therefore no need. Thanks!